File-Formats/DocIO/Convert-Word-Document-to-PDF-in-AWS.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Convert Word to PDF in AWS | Syncfusion
+description: Learn how to convert a Word document to a PDF in Amazon Web Services (AWS) using Syncfusion .NET Core Word (DocIO) library in C#.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to PDF in Amazon Web Services (AWS)
+
+Syncfusion DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, **convert a Word document to PDF in Amazon Web Services (AWS)** within a few lines of code. 
+
+N> If this is your first time working with Amazon Web Services (AWS), please refer to the dedicated AWS resources. This section explains how to convert Word documents to PDF in C# using the .NET Core Word (DocIO) library in AWS. 
+
+## Prerequisites 
+
+* An active **Amazon Web Services (AWS) account** is required. If you don’t have one, please [create a account](https://aws.amazon.com/) before starting.
```

Comment:
An active Amazon Web Services (AWS) account is required. If you don’t have one, please [create an account](https://aws.amazon.com/) before starting.

Address the same grammar "an account" for Azure home page also

Reply:
Updated